org-code-javabuilder/media/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    // Apply the java-library plugin for API and implementation separation.
+    id 'java-library'
```

Comment:
Also add `id 'com.github.sherter.google-java-format' version '0.8'` to enable the linter.